### ISSUE TITLE
Error in files with corrupted or unsupported video mimetype

### ISSUE THREAD

**Issue by 1581-netizen** [Nekmo/telegram-upload#82]
* telegram-upload version: 0.3.4 
* Python version: 3.7
* Operating System: Ubuntu
* Dependencies list (run `pip freeze`): i have all dependencies 

I have try to upload .ts file but not working 

### What I Did

```
telegram-upload --no-thumbnail --to --caption "" "/docker/te/test.ts"
Error:

Traceback (most recent call last):
  File "/usr/local/bin/telegram-upload", line 8, in <module>
    sys.exit(upload_cli())
  File "/usr/local/lib/python3.7/dist-packages/telegram_upload/exceptions.py", line 46, in wrap
    fn(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.7/dist-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.7/dist-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/telegram_upload/management.py", line 45, in upload
    client.send_files(to, files, delete_on_success, print_file_id, force_file, forward, caption, no_thumbnail)
  File "/usr/local/lib/python3.7/dist-packages/telegram_upload/client.py", line 80, in send_files
    attributes = get_file_attributes(file)
  File "/usr/local/lib/python3.7/dist-packages/telegram_upload/files.py", line 30, in get_file_attributes
    if not metadata.has('width') and meta_groups:
AttributeError: 'NoneType' object has no attribute 'has'

```

**Comment by Nekmo** [comment]
Can you get the mimetype from typescript files on your machine? Open a python console:

```
$ python
```

And type:
```
import mimetypes

mimetypes.init()

print(mimetypes.guess_type('foo.ts'))
```

For example:

```
$ python
Python 3.6.11 (default, Aug  4 2020, 23:09:21) 
[GCC 10.1.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import mimetypes
>>> 
>>> mimetypes.init()
>>> 
>>> print(mimetypes.guess_type('foo.ts'))
('text/vnd.trolltech.linguist', None)
>>> 
```

**Comment by 1581-netizen** [comment]
Hi Nekmo, 

('video/MP2T', None)

**Comment by Nekmo** [comment]
You can fix the error by changing the system mimetype.

https://stackoverflow.com/questions/14943742/how-to-add-new-mime-type-and-associate-with-new-application

Anyway I will fix it in a future release, thanks!

**Comment by Nekmo** [comment]
Fixed in v0.5.0.
